cli/crates/typed-resolvers/src/lib.rs

```diff
@@ -62,7 +64,7 @@ pub fn check_resolvers(resolvers_root: &Path, schema: &analyze::AnalyzedSchema<'
     AnalyzedResolvers { errs }
 }
 
-fn experimental_codegen_is_enabled(parsed_schema: &engine_parser::types::ServiceDocument) -> bool {
+pub fn experimental_codegen_is_enabled(parsed_schema: &engine_parser::types::ServiceDocument) -> bool {
```

Comment:
Technically, we could move this to the registry now, but this works and it will go away in the not too far away future, so I decided not to make the change.